apps/OboeTester/app/src/main/res/layout/activity_main.xml

```diff
@@ -182,42 +182,15 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toEndOf="@+id/textView" />
 
-    <LinearLayout
-        android:id="@+id/layoutBluetoothSco"
```

Comment:
We may want to just test swapping between SCO and A2DP outputs. We won't be able to test this if we move this into input activities.

Reply:
TEST OUTPUT does have the UI.
We should add it to TapToTone, etc.